### PR TITLE
deb,rpm: fix python-cephfs dependencies

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
https://shaman.ceph.com/builds/ceph/wip-python-cephfs-dependencies/

**Comment by tchaikov** [comment]
retest this please.

**Comment by smithfarm** [comment]
failure in `run-rbd-unit-tests.sh` appears to be transient

jenkins re-test this please

File: debian/control [hunk 1]
```diff
@@ -991,6 +996,8 @@ Package: python3-cephfs
 Architecture: linux-any
 Section: python
 Depends: libcephfs2 (= ${binary:Version}),
+         python3-rados (= ${binary:Version}),
+         python-ceph-argparse (= ${binary:Version}),
```
Comment (smithfarm) [inline debian/control]: does this mean python3-cephfs requires python2 on debian?

**Comment by smithfarm** [comment]
RPM side looks good to me

File: debian/control (hunk 1 above)
Comment (tchaikov) [reply debian/control]: @smithfarm thanks for reviewing this change. that's was an oversight. 

fixed and repushed.

**Comment by tchaikov** [comment]
should run this with an upgrade test,  "luminous-x" at least.

**Comment by liewegas** [comment]
```
2018-07-17T16:11:56.720 INFO:teuthology.orchestra.run.smithi082.stdout:The following packages have unmet dependencies:
2018-07-17T16:11:56.771 INFO:teuthology.orchestra.run.smithi082.stdout: ceph-common : Depends: python-cephfs (= 14.0.0-1366-g6ed10b3-1bionic) but it is not going to be installed
2018-07-17T16:11:56.771 INFO:teuthology.orchestra.run.smithi082.stdout:               Depends: python-ceph-argparse (= 14.0.0-1366-g6ed10b3-1bionic) but it is not installable
2018-07-17T16:11:56.775 INFO:teuthology.orchestra.run.smithi082.stdout: python-ceph : Depends: python-cephfs (= 14.0.0-1366-g6ed10b3-1bionic) but it is not going to be installed
```
debs not working... /a/sage-2018-07-17_14:21:13-rados-wip-sage3-testing-2018-07-17-0720-distro-basic-smithi/2791184

**Comment by tchaikov** [comment]
will re-schedule a test run once https://shaman.ceph.com/builds/ceph/wip-23043-kefu/8a0069e5d6357c5324f8d21adfb9bbe01a66e70c/ is done.

**Comment by batrick** [comment]
@tchaikov status on this?

**Comment by tchaikov** [comment]
@batrick http://pulpito.ceph.com/kchai-2018-07-25_02:04:25-rados-wip-23043-kefu-distro-basic-mira/

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-07-25_15:31:23-rados-wip-23043-kefu-distro-basic-smithi

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-07-26_04:15:27-rados-wip-kefu-testing-2018-07-26-0913-distro-basic-mira/

**Comment by batrick** [comment]
http://pulpito.ceph.com/pdonnell-2018-07-27_22:41:15-fs-wip-pdonnell-testing-20180727.210741-distro-basic-smithi/

**Pull request merged**